opentelemetry-semantic-conventions/scripts/templates/registry/rust/metric.rs.j2

```diff
@@ -0,0 +1,79 @@
+{%- import 'macros.j2' as metric_macros -%}
+// DO NOT EDIT, this is an auto-generated file
+//
+// If you want to update the file:
+// - Edit the template at scripts{{template}}
+// - Run the script at scripts/generate-consts-from-spec.sh
+
+//! # Metric Semantic Conventions
+//!
+//! The [metric semantic conventions] define a set of standardized attributes to
+//! be used in `Meter`s.
+//!
+//! [metric semantic conventions]: https://github.com/open-telemetry/semantic-conventions/tree/main/model/metric
+//!
+//! ## Usage
+//!
+//! ```rust
+//! use opentelemetry::{global, KeyValue};
+//! use opentelemetry_semantic_conventions as semconv;
+//!
+//! // Assumes we already have an initialized `MeterProvider`
+//! // See: https://github.com/open-telemetry/opentelemetry-rust/blob/main/examples/metrics-basic/src/main.rs
+//! // for an example
+//! let meter = global::meter("mylibraryname");
+//! let histogram = meter
+//!     .u64_histogram(semconv::metric::HTTP_SERVER_REQUEST_DURATION)
+//!     .with_unit("By")
+//!     .with_description("Duration of HTTP server requests.")
+//!     .init();
+//! ```
+
+{% for root_ns in ctx %}
+    {% for metric in root_ns.metrics %}
+{{ ["## Description\n\n", metric.brief, concat_if("\n\n# Notes\n\n", metric.note), metric_macros.examples(metric)] | comment }}
+/// ## Metadata
+/// | | |
+/// |:-|:-
+/// | Instrument: | `{{ metric.instrument }}` |
+/// | Unit: | `{{ metric.unit }}` |
+/// | Status: | `{{ metric.stability | capitalize }}`  |
+        {% if metric.attributes %}
+///
+/// ## Attributes
+/// | Name | Requirement |
+/// |:-|:- |
+        {% endif %}
+        {% for attribute in metric.attributes | rejectattr("name", "in", params.excluded_attributes) | sort(attribute="name") %}
+            {% if attribute.requirement_level %}
+                {% if attribute.requirement_level.conditionally_required %}
+                    {% set req_level = "Conditionally_required" %}
+                    {% set req_message = attribute.requirement_level.conditionally_required %}
+                {% else %}
+                    {% set req_level = (attribute.requirement_level | capitalize) %}
+                    {% set req_message = attribute.requirement_level_msg %}
+                {% endif %}
+            {% else %}
+                {% set req_level = "Unspecified" %}
+                {% set req_message = '' %}
+            {% endif %}
+/// | [`crate::attribute::{{ attribute.name | screaming_snake_case }}`] | `{{ req_level }}`{{ (': ' + req_message if req_message else '') }}
+        {% endfor %}
+        {% if metric.examples %}
+///
+/// # Examples
```

Comment:
Same as above:
```suggestion
/// ## Examples
```

Reply:
Fixed in [4970ee9](https://github.com/open-telemetry/opentelemetry-rust/pull/2098/commits/4970ee9314ca5cb72376b21bb807ec14da0d4cc8)